### ISSUE TITLE
Adjust the size of the WQE for "monte carlo" request

### ISSUE THREAD

**Issue by vlimant** [dmwm/WMCore#8060]
looking at the failure for 

https://cmsweb.cern.ch/reqmgr2/fetch?rid=mcremone_task_TOP-RunIIWinter15wmLHE-00061__v1_T_170809_194110_9524

it looks like the size of the WQE should be smaller so as to not end up with a humengous amount of lumis in it.

**Comment by ticoann** [comment]
We can control this 
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WorkQueue/Policy/Start/MonteCarlo.py#L31

That can be passed by setting config below. 

https://github.com/dmwm/deployment/blob/master/workqueue/config.py#L76
```
config.queueParams['SplittingMapping'] = {'MonteCarlo':  {'name': 'MonteCarlo',
                                                                'args': {'MaxJobsPerElement', 100}}
https://github.com/dmwm/deployment/blob/master/workqueue/config.py#L76
```

However, this will change all the MC settings. I think we have to find the solution automatically adjust number of jobs when lumi count is too big for given WQE.

@amaltaro, Alan do you remember what was the exact problem when there are too many lumis in WQE?

**Comment by amaltaro** [comment]
We used to have this value set to 250 jobs per element (MC type). The problem with that was with these very low efficiency workflows, where reqmgr "compensates" the eff < 1 with more events (such that it can deliver the amount of events requested).

That means the workflow can become huge wqe-wise, and some operations like force-completing or aborting a workflow may take minutes to complete. The WorkQueueManager performance is also affected, since it has to go through more of those elements.

Perhaps we could try lowering it to 750 jobs (or even 500)? Anyways, I'm pretty sure this issue will be back in a couple of months where we need to do something else.

Ideally, we should change this design such that we store a range of lumis per row, instead a single lumi per row.

Shall I ask Lina to update the workqueue config in testbed? I guess we go directly with 500?

**Comment by amaltaro** [comment]
@ticoann given you configuration example, I have decreased the WQE size to 600 jobs
https://github.com/dmwm/deployment/pull/530

if you agree, then we can push this change in in the next upgrade. My only concern is aborting or force-completing a very big workflow. Thats a synchronous operation, so the client only gets the response back after all the elements have been updated in couch. We _just_ need to watch out for these cases...
